pages/common/sass.md

```diff
@@ -1,6 +1,7 @@
 # sass
 
 > Converts SCSS or Sass files to CSS.
+> Homepage: <https://sass-lang.com>.
```

Comment:
According to https://sass-lang.com/documentation/cli, there are two CLIs for Sass: `dart-sass` and `ruby-sass`, both being invoked with simply `sass`. Since the ruby CLI is deprecated, I'd say we should link directly to the dart-sass page:

```suggestion
> Homepage: <https://sass-lang.com/documentation/cli/dart-sass>.
```